packages/core/src/components/modal/modal.scss

```diff
@@ -0,0 +1,101 @@
+/* Sass mixin defined to handle one and two buttons */
+@mixin child-count($num) {
+    @if $num == 1 {
+        &:only-child {
+            @content;
+        }
+    }
+    @else {
+        @content
+    }
+}
```

Comment:
I don't think we should specialize on the number if buttons, let's leave that to the user. For context, take a look at the card component. It provides a slot where users can add buttons or other components that serve as "actions"